projects/plugins/jetpack/modules/subscriptions/subscribe-floating-button/class-jetpack-subscribe-floating-button.php

```diff
@@ -0,0 +1,193 @@
+<?php
+/**
+ * Adds support for Jetpack floating Subscribe button feature
+ *
+ * @package automattic/jetpack-subscriptions
+ * @since $$next-version$$
+ */
+
+/**
+ * Jetpack_Subscribe_Floating_Button class.
+ */
+class Jetpack_Subscribe_Floating_Button {
+	/**
+	 * Jetpack_Subscribe_Floating_Button singleton instance.
+	 *
+	 * @var Jetpack_Subscribe_Floating_Button|null
+	 */
+	private static $instance;
+
+	/**
+	 * Jetpack_Subscribe_Floating_Button instance init.
+	 */
+	public static function init() {
+		if ( self::$instance === null ) {
+			self::$instance = new Jetpack_Subscribe_Floating_Button();
+		}
+
+		return self::$instance;
+	}
+
+	const BLOCK_TEMPLATE_PART_SLUG = 'jetpack-subscribe-floating-button';
+
+	/**
+	 * Jetpack_Subscribe_Floating_Button class constructor.
+	 */
+	public function __construct() {
+		if ( get_option( 'jetpack_subscribe_floating_button_enabled', true ) ) {
```

Comment:
Let's disable it by default.

```suggestion
		if ( get_option( 'jetpack_subscribe_floating_button_enabled', false ) ) {
```